infra/modules/identity-provider-client/main.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_cognito_user_pool_client" "client" {
```

Comment:
```suggestion
# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cognito_user_pool_client
resource "aws_cognito_user_pool_client" "client" {
```

This is something I like to do, especially for complex terraform resources. I still remember years ago being a terraform newbie and not knowing how to connect the configuration to the documentation